### DISCUSSION TITLE
Hayward AquaRite Settings changing when remotely controlled

### DISCUSSION THREAD

**Discussion by Fumado** [tagyoureit/nodejs-poolController#981] [App and O/S Config Questions]
So I've been using NJSPC for several years now, and I have version 7.7.  It's been working fine, so no need to upgrade, eh?  

I have an AQR-15 generator and cleaned the salt cell last weekend because the salt level was down, and that's usually an indication that it needs to be cleaned.  I held the diagnostic button until I heard a click of the relay, and maybe this did some reset to the unit instead of clearing the inspect cell light.

The next day, all of the readings were screwy, the salt level was 0, or it was high-salt and it was not generating.  The display screen was flashing through all the parameters as if you were touching the diagnostic button to advance the parameters.  Also the salt cell type was changing from T-3 to T-15 and whatever else there was.  

I disconnected the power to the remote interface and everything calmed down.  The remote number was AL-5 instead of the  default AL-0.

So the question is, I'm now reluctant to reconnect the NJSPC, until I'm confident that it won't disturb any settings.  What is the correct sequence to reinitialize it with the AquaRite?

**Comment by tagyoureit** [comment]
Regardless of your njsPC version, you should fix the hardware errors first.  There's nothing our software can do to overcome an erratic/broken salt cell. 

Edit: I wasn't clear if you wanted to keep the salt cell connected or not; if you want to try the software without it... it's been so long since 7.7 I can't really recall what the steps are to remove a chlorinator.  I'd start with a new set up (keep
Your current install as backup if you want) and all of the instructions can be found in the wiki.
